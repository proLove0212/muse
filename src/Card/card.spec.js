import test from '../utils/testing';
import Card from './card';

test('Card', ({ mount }) => {
  it('should card view', async () => {
    const wrapper = mount(Card, {});
    expect(wrapper.html()).toMatchSnapshot();
    wrapper.setProps({
      raised: true
    });
    expect(wrapper.html()).toMatchSnapshot();
  });
});
