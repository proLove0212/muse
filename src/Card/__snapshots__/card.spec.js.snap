// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card should card view 1`] = `

<div class="mu-card">
</div>

`;

exports[`Card should card view 2`] = `

<div class="mu-card mu-card__raised">
</div>

`;
